### ISSUE TITLE
vae enc_dec  configuration questions

### ISSUE THREAD

**Issue by treasuraid** [lucidrains/muse-maskgit-pytorch#24]
I had a question on model configurations with paper.

In paper, f=8 thus the for super-res model's latent map has 64*64 resolution map with image size 512*512.

Thus, I was trying with the vae with number of layer is 3 thus to make sure they have 64*64 resolution. However I think it may different with paper's implementation detail that made vae consisit of 4 layers and 256 dim with finetuning or 2 layers and 128 dim without finetuning decoder.

How can I set configuration of Vae in code to match with paper's?
